.github/workflows/test.yml

```diff
@@ -15,20 +15,18 @@ jobs:
       - name: Install Ruby ${{ matrix.ruby }}
         uses: ruby/setup-ruby@v1
         with:
-          ruby-version: "3.1"
+          # run rubocop with the ruby version we lint for
+          ruby-version: "2.7"
```

Comment:
Why? Normally you set target version in RuboCop (which you did) so why does it matter?